ocaml/xapi/xapi_vdi.ml

```diff
@@ -25,6 +25,30 @@ open Printf
 (**************************************************************************************)
 (* current/allowed operations checking                                                *)
 
+let check_sm_feature_error op sm_features sr =
```

Comment:
I think we could move this `let` into the `check_operation_error` function, to make it more local, as it's not used anywhere else. I usually prefer making declarations as local as possible, that clarifies where each declaration is used, gives code a better structure, and keeps related code together.